docs/understand/roadmap.md

```diff
@@ -14,135 +14,145 @@ redirect_from: ../roadmap.html
+- Resilience: Since every Spark write is a commit, it is also undoable by reverting it.
 
-For Delta Lake in particular, these metadata files represent a [logical transaction log that relies on numerical ordering](https://github.com/delta-io/delta/blob/master/PROTOCOL.md#delta-log-entries).
+[Track and discuss on GitHub](https://github.com/treeverse/lakeFS/blob/master/design/open/spark-outputcommitter/committer.md){: target="_blank" class="btn" }
 
-Currently, when trying to modify a Delta table from 2 different branches, lakeFS would correctly recognize a conflict: this log diverged into 2 different copies, representing different changes.
-Users would then have to forgo one of the change sets, by either retaining the destination's branch set of changes, or the source's branch.
 
-A much better user experience would be to allow merging this log into a new unified set of changes, representing changes made in both branches as a new set of log files (and potentially, data files too!).
+### Native connector: Trino
 
-While beneficial for Delta Lake users, this is a departure from the un-opinionated nature of lakeFS, that is kept simple by treating objects as opaque blobs.
-We're still gathering information on the use cases and access patterns where this is beneficial - please let us know if this is something you'd find interesting
+Currently, the Trino integration works well using the [lakeFS s3 Gateway](architecture.md#s3-gateway). 
 
+While easy to integrate and useful out of the box, due to the S3 protocol, it means that the data itself must pass through the lakeFS server.
 
-[Contact us, we'd love to talk about it!](mailto:hello@treeverse.io?subject=using+lakeFS+with+Delta+Lake){: target="_blank" class="btn" }
+For larger installations, a native integration where lakeFS handles metadata, returning locations in the underlying object store that Trino can then access directly, would allow reducing the operational overhead and increasing the scalability of lakeFS.
+This would be done in a similar way to the [Native Spark integration](../integrations/spark.md) using the [Hadoop Filesystem implementation](../integrations/spark.md#access-lakefs-using-the-lakefs-specific-hadoop-filesystem).
 
+[Track and discuss on GitHub](https://github.com/treeverse/lakeFS/issues/2357){: target="_blank" class="btn" }
 
 
-### Hooks: usability improvements <span>High Priority</span>{: .label }
+### Improved streaming support for Apache Kafka
 
-While hooks are an immensely useful tool that provides strong guarantees to data consumers, we want to make them more useful but also easier to implement:
+Committing (along with attaching useful information to the commit) makes a lot of sense for batch workloads: 
+- run a job or a pipeline on a separate branch and commit 
+- record information such as the git hash of the code executed, the versions of frameworks used, and information about the data artifacts
```

Comment:
Topic offset is a great metadata info to add, maybe mention it explicitly?

Reply:
This is listed explicitly in line #104 :)